### PR TITLE
UFS-dev PR#98

### PR DESCRIPTION
**Pull request by grantfirl**
Identical to https://github.com/ufs-community/ufs-weather-model/pull/1876

Also includes:
https://github.com/ufs-community/ufs-weather-model/pull/1707 (BL changes): Anticipation is that the majority of RTs will change as this is a fundamental stack change. New baselines will most likely be required.

https://github.com/ufs-community/ufs-weather-model/pull/1773 (BL changes):
139:Test 001 cpld_control_p8_mixedmode_intel FAIL
210:Test 002 cpld_control_gfsv17_intel FAIL
282:Test 003 cpld_control_p8_intel FAIL
354:Test 005 cpld_control_qr_p8_intel FAIL
414:Test 007 cpld_2threads_p8_intel FAIL
474:Test 008 cpld_decomp_p8_intel FAIL
534:Test 009 cpld_mpi_p8_intel FAIL
606:Test 010 cpld_control_ciceC_p8_intel FAIL
666:Test 011 cpld_control_c192_p8_intel FAIL
721:Test 013 cpld_bmark_p8_intel FAIL
792:Test 015 cpld_control_noaero_p8_intel FAIL
861:Test 016 cpld_control_nowave_noaero_p8_intel FAIL
921:Test 017 cpld_debug_p8_intel FAIL
980:Test 018 cpld_debug_noaero_p8_intel FAIL
1049:Test 019 cpld_control_noaero_p8_agrid_intel FAIL
1106:Test 020 cpld_control_c48_intel FAIL
1178:Test 021 cpld_control_p8_faster_intel FAIL
1249:Test 022 cpld_control_pdlib_p8_intel FAIL
1308:Test 025 cpld_debug_pdlib_p8_intel FAIL
4901:Test 158 datm_cdeps_control_cfsr_intel FAIL
4914:Test 160 datm_cdeps_control_gefs_intel FAIL
4927:Test 161 datm_cdeps_iau_gefs_intel FAIL
4940:Test 162 datm_cdeps_stochy_gefs_intel FAIL
4953:Test 163 datm_cdeps_ciceC_cfsr_intel FAIL
4966:Test 164 datm_cdeps_bulk_cfsr_intel FAIL
4979:Test 165 datm_cdeps_bulk_gefs_intel FAIL
4995:Test 166 datm_cdeps_mx025_cfsr_intel FAIL
5011:Test 167 datm_cdeps_mx025_gefs_intel FAIL
5022:Test 168 datm_cdeps_multiple_files_cfsr_intel FAIL
5035:Test 169 datm_cdeps_3072x1536_cfsr_intel FAIL
5048:Test 170 datm_cdeps_gfs_intel FAIL
5061:Test 171 datm_cdeps_debug_cfsr_intel FAIL
5074:Test 172 datm_cdeps_control_cfsr_faster_intel FAIL
5248:Test 176 atmwav_control_noaero_p8_intel FAIL
7330:Test 244 cpld_control_p8_gnu FAIL
7399:Test 245 cpld_control_nowave_noaero_p8_gnu FAIL
7459:Test 246 cpld_debug_p8_gnu FAIL
7530:Test 247 cpld_control_pdlib_p8_gnu FAIL
7589:Test 248 cpld_debug_pdlib_p8_gnu FAIL
7602:Test 249 datm_cdeps_control_cfsr_gnu FAIL

https://github.com/ufs-community/ufs-weather-model/pull/1880 (BL changes): All tests with "conus13km" in their name are replaced by new ones with shorter names. New tests hrrr_gf and hrrr_c3 are added.

https://github.com/ufs-community/ufs-weather-model/pull/1467 (BL changes)

https://github.com/ufs-community/ufs-weather-model/pull/1871 (input data change):
The input data in WW3_input_data_20220624 which is on hera needs to be copied to the WCOSS2 input locations for the extra files needed for the pdlib cases to be turned on.

https://github.com/ufs-community/ufs-weather-model/pull/1870 (no BL change)

https://github.com/ufs-community/ufs-weather-model/pull/1823 (no BL change)

### CODE REVIEW

**Comment by mkavulich** [comment]
@grantfirl That's frustrating the permissions are still bad. I tried updating umask in my `.bashrc` file so we'll see if that helps next time. In the meantime I moved the new baseline into place and started these tests.

**Comment by mkavulich** [comment]
***Automated RT Failure Notification***
Machine: hera
Compiler: intel
Job: RT
[RT] Repo location: /scratch1/BMC/gmtb/CCPP_regression_testing/NCAR_ufs-weather-model//run//1585301644/20240117014515/ufs-weather-model
[RT] Error: Test 001 cpld_control_p8_mixedmode_intel FAIL Tries: 2
[RT] Error: Test 002 cpld_control_gfsv17_intel FAIL Tries: 2
[RT] Error: Test 003 cpld_control_p8_intel FAIL Tries: 2
[RT] Error: Test 005 cpld_control_qr_p8_intel FAIL Tries: 2
[RT] Error: Test 007 cpld_2threads_p8_intel FAIL Tries: 2
[RT] Error: Test 008 cpld_decomp_p8_intel FAIL Tries: 2
[RT] Error: Test 009 cpld_mpi_p8_intel FAIL Tries: 2
[RT] Error: Test 010 cpld_control_ciceC_p8_intel FAIL Tries: 2
[RT] Error: Test 011 cpld_control_c192_p8_intel FAIL Tries: 2
[RT] Error: Test 013 cpld_bmark_p8_intel FAIL Tries: 2
[RT] Error: Test 015 cpld_control_noaero_p8_intel FAIL Tries: 2
[RT] Error: Test 016 cpld_control_nowave_noaero_p8_intel FAIL Tries: 2
[RT] Error: Test 017 cpld_debug_p8_intel FAIL Tries: 2
[RT] Error: Test 018 cpld_debug_noaero_p8_intel FAIL Tries: 2
[RT] Error: Test 019 cpld_control_noaero_p8_agrid_intel FAIL Tries: 2
[RT] Error: Test 020 cpld_control_c48_intel FAIL Tries: 2
[RT] Error: Test 021 cpld_control_p8_faster_intel FAIL Tries: 2
[RT] Error: Test 022 cpld_control_pdlib_p8_intel FAIL Tries: 2
[RT] Error: Test 025 cpld_debug_pdlib_p8_intel FAIL Tries: 2
[RT] Error: Test 026 control_flake_intel FAIL Tries: 2
[RT] Error: Test 027 control_CubedSphereGrid_intel FAIL Tries: 2
[RT] Error: Test 028 control_CubedSphereGrid_parallel_intel FAIL Tries: 2
[RT] Error: Test 029 control_latlon_intel FAIL Tries: 2
[RT] Error: Test 031 control_c48_intel FAIL Tries: 2
[RT] Error: Test 032 control_c192_intel FAIL Tries: 2
[RT] Error: Test 033 control_c384_intel FAIL Tries: 2
[RT] Error: Test 034 control_c384gdas_intel FAIL Tries: 2
[RT] Error: Test 035 control_stochy_intel FAIL Tries: 2
[RT] Error: Test 037 control_lndp_intel FAIL Tries: 2
[RT] Error: Test 038 control_iovr4_intel FAIL Tries: 2
[RT] Error: Test 039 control_iovr5_intel FAIL Tries: 2
[RT] Error: Test 040 control_p8_intel FAIL Tries: 2
[RT] Error: Test 042 control_qr_p8_intel FAIL Tries: 2
[RT] Error: Test 044 control_decomp_p8_intel FAIL Tries: 2
[RT] Error: Test 045 control_2threads_p8_intel FAIL Tries: 2
[RT] Error: Test 046 control_p8_lndp_intel FAIL Tries: 2
[RT] Error: Test 047 control_p8_rrtmgp_intel FAIL Tries: 2
[RT] Error: Test 048 control_p8_mynn_intel FAIL Tries: 2
[RT] Error: Test 049 merra2_thompson_intel FAIL Tries: 2
[RT] Error: Test 050 regional_control_intel FAIL Tries: 2
[RT] Error: Test 052 regional_control_qr_intel FAIL Tries: 2
[RT] Error: Test 054 regional_decomp_intel FAIL Tries: 2
[RT] Error: Test 055 regional_2threads_intel FAIL Tries: 2
[RT] Error: Test 057 regional_netcdf_parallel_intel FAIL Tries: 2
[RT] Error: Test 058 regional_2dwrtdecomp_intel FAIL Tries: 2
[RT] Error: Test 059 regional_wofs_intel FAIL Tries: 2
[RT] Error: Test 060 rap_control_intel FAIL Tries: 2
[RT] Error: Test 061 regional_spp_sppt_shum_skeb_intel FAIL Tries: 2
[RT] Error: Test 062 rap_decomp_intel FAIL Tries: 2
[RT] Error: Test 063 rap_2threads_intel FAIL Tries: 2
[RT] Error: Test 065 rap_sfcdiff_intel FAIL Tries: 2
[RT] Error: Test 066 rap_sfcdiff_decomp_intel FAIL Tries: 2
[RT] Error: Test 068 hrrr_control_intel FAIL Tries: 2
[RT] Error: Test 069 hrrr_control_qr_intel FAIL Tries: 2
[RT] Error: Test 070 hrrr_control_decomp_intel FAIL Tries: 2
[RT] Error: Test 071 hrrr_control_2threads_intel FAIL Tries: 2
[RT] Error: Test 074 rrfs_v1beta_intel FAIL Tries: 2
[RT] Error: Test 075 rrfs_v1nssl_intel FAIL Tries: 2
[RT] Error: Test 076 rrfs_v1nssl_nohailnoccn_intel FAIL Tries: 2
[RT] Error: Test 077 hrrr_gf_intel FAIL Tries: 2
[RT] Error: Test 078 hrrr_c3_intel FAIL Tries: 2
[RT] Error: Test 079 control_csawmg_intel FAIL Tries: 2
[RT] Error: Test 080 control_csawmgt_intel FAIL Tries: 2
[RT] Error: Test 081 control_ras_intel FAIL Tries: 2
[RT] Error: Test 082 control_wam_intel FAIL Tries: 2
[RT] Error: Test 083 control_p8_faster_intel FAIL Tries: 2
[RT] Error: Test 084 regional_control_faster_intel FAIL Tries: 2
[RT] Error: Test 085 control_CubedSphereGrid_debug_intel FAIL Tries: 2
[RT] Error: Test 086 control_wrtGauss_netcdf_parallel_debug_intel FAIL Tries: 2
[RT] Error: Test 087 control_stochy_debug_intel FAIL Tries: 2
[RT] Error: Test 088 control_lndp_debug_intel FAIL Tries: 2
[RT] Error: Test 089 control_csawmg_debug_intel FAIL Tries: 2
[RT] Error: Test 090 control_csawmgt_debug_intel FAIL Tries: 2
[RT] Error: Test 091 control_ras_debug_intel FAIL Tries: 2
[RT] Error: Test 092 control_diag_debug_intel FAIL Tries: 2
[RT] Error: Test 093 control_debug_p8_intel FAIL Tries: 2
[RT] Error: Test 094 regional_debug_intel FAIL Tries: 2
[RT] Error: Test 095 rap_control_debug_intel FAIL Tries: 2
[RT] Error: Test 096 hrrr_control_debug_intel FAIL Tries: 2
[RT] Error: Test 097 rap_unified_drag_suite_debug_intel FAIL Tries: 2
[RT] Error: Test 098 rap_diag_debug_intel FAIL Tries: 2
[RT] Error: Test 099 rap_cires_ugwp_debug_intel FAIL Tries: 2
[RT] Error: Test 100 rap_unified_ugwp_debug_intel FAIL Tries: 2
[RT] Error: Test 101 rap_lndp_debug_intel FAIL Tries: 2
[RT] Error: Test 102 rap_progcld_thompson_debug_intel FAIL Tries: 2
[RT] Error: Test 103 rap_noah_debug_intel FAIL Tries: 2
[RT] Error: Test 104 rap_sfcdiff_debug_intel FAIL Tries: 2
[RT] Error: Test 105 rap_noah_sfcdiff_cires_ugwp_debug_intel FAIL Tries: 2
[RT] Error: Test 106 rrfs_v1beta_debug_intel FAIL Tries: 2
[RT] Error: Test 107 rap_clm_lake_debug_intel FAIL Tries: 2
[RT] Error: Test 108 rap_flake_debug_intel FAIL Tries: 2
[RT] Error: Test 109 control_wam_debug_intel FAIL Tries: 2
[RT] Error: Test 110 regional_spp_sppt_shum_skeb_dyn32_phy32_intel FAIL Tries: 2
[RT] Error: Test 111 rap_control_dyn32_phy32_intel FAIL Tries: 2
[RT] Error: Test 112 hrrr_control_dyn32_phy32_intel FAIL Tries: 2
[RT] Error: Test 113 hrrr_control_qr_dyn32_phy32_intel FAIL Tries: 2
[RT] Error: Test 114 rap_2threads_dyn32_phy32_intel FAIL Tries: 2
[RT] Error: Test 115 hrrr_control_2threads_dyn32_phy32_intel FAIL Tries: 2
[RT] Error: Test 116 hrrr_control_decomp_dyn32_phy32_intel FAIL Tries: 2
[RT] Error: Test 120 conus13km_control_intel FAIL Tries: 2
[RT] Error: Test 123 rap_control_dyn64_phy32_intel FAIL Tries: 2
[RT] Error: Test 124 rap_control_debug_dyn32_phy32_intel FAIL Tries: 2
[RT] Error: Test 125 hrrr_control_debug_dyn32_phy32_intel FAIL Tries: 2
[RT] Error: Test 126 conus13km_debug_intel FAIL Tries: 2
[RT] Error: Test 127 conus13km_debug_2threads_intel FAIL Tries: 2
[RT] Error: Test 128 conus13km_radar_tten_debug_intel FAIL Tries: 2
[RT] Error: Test 129 rap_control_dyn64_phy32_debug_intel FAIL Tries: 2
[RT] Error: Test 130 hafs_regional_atm_intel FAIL Tries: 2
[RT] Error: Test 131 hafs_regional_atm_thompson_gfdlsf_intel FAIL Tries: 2
[RT] Error: Test 132 hafs_regional_atm_ocn_intel FAIL Tries: 2
[RT] Error: Test 133 hafs_regional_atm_wav_intel FAIL Tries: 2
[RT] Error: Test 134 hafs_regional_atm_ocn_wav_intel FAIL Tries: 2
[RT] Error: Test 135 hafs_regional_1nest_atm_intel FAIL Tries: 2
[RT] Error: Test 136 hafs_regional_1nest_atm_qr_intel FAIL Tries: 2
[RT] Error: Test 137 hafs_regional_telescopic_2nests_atm_intel FAIL Tries: 2
[RT] Error: Test 138 hafs_global_1nest_atm_intel FAIL Tries: 2
[RT] Error: Test 139 hafs_global_1nest_atm_qr_intel FAIL Tries: 2
[RT] Error: Test 140 hafs_global_multiple_4nests_atm_intel FAIL Tries: 2
[RT] Error: Test 141 hafs_global_multiple_4nests_atm_qr_intel FAIL Tries: 2
[RT] Error: Test 142 hafs_regional_specified_moving_1nest_atm_intel FAIL Tries: 2
[RT] Error: Test 143 hafs_regional_storm_following_1nest_atm_intel FAIL Tries: 2
[RT] Error: Test 144 hafs_regional_storm_following_1nest_atm_qr_intel FAIL Tries: 2
[RT] Error: Test 145 hafs_regional_storm_following_1nest_atm_ocn_intel FAIL Tries: 2
[RT] Error: Test 146 hafs_global_storm_following_1nest_atm_intel FAIL Tries: 2
[RT] Error: Test 147 hafs_regional_storm_following_1nest_atm_ocn_debug_intel FAIL Tries: 2
[RT] Error: Test 148 hafs_regional_storm_following_1nest_atm_ocn_wav_intel FAIL Tries: 2
[RT] Error: Test 149 hafs_regional_docn_intel FAIL Tries: 2
[RT] Error: Test 150 hafs_regional_docn_oisst_intel FAIL Tries: 2
[RT] Error: Test 151 hafs_regional_datm_cdeps_intel FAIL Tries: 2
[RT] Error: Test 152 datm_cdeps_control_cfsr_intel FAIL Tries: 2
[RT] Error: Test 154 datm_cdeps_control_gefs_intel FAIL Tries: 2
[RT] Error: Test 155 datm_cdeps_iau_gefs_intel FAIL Tries: 2
[RT] Error: Test 156 datm_cdeps_stochy_gefs_intel FAIL Tries: 2
[RT] Error: Test 157 datm_cdeps_ciceC_cfsr_intel FAIL Tries: 2
[RT] Error: Test 158 datm_cdeps_bulk_cfsr_intel FAIL Tries: 2
[RT] Error: Test 159 datm_cdeps_bulk_gefs_intel FAIL Tries: 2
[RT] Error: Test 160 datm_cdeps_mx025_cfsr_intel FAIL Tries: 2
[RT] Error: Test 161 datm_cdeps_mx025_gefs_intel FAIL Tries: 2
[RT] Error: Test 162 datm_cdeps_multiple_files_cfsr_intel FAIL Tries: 2
[RT] Error: Test 163 datm_cdeps_3072x1536_cfsr_intel FAIL Tries: 2
[RT] Error: Test 164 datm_cdeps_gfs_intel FAIL Tries: 2
[RT] Error: Test 165 datm_cdeps_debug_cfsr_intel FAIL Tries: 2
[RT] Error: Test 166 datm_cdeps_control_cfsr_faster_intel FAIL Tries: 2
[RT] Error: Test 167 datm_cdeps_lnd_gswp3_intel FAIL Tries: 2
[RT] Error: Test 169 control_p8_atmlnd_sbs_intel FAIL Tries: 2
[RT] Error: Test 170 atmwav_control_noaero_p8_intel FAIL Tries: 2
[RT] Error: Test 171 control_atmwav_intel FAIL Tries: 2
[RT] Error: Test 172 atmaero_control_p8_intel FAIL Tries: 2
[RT] Error: Test 173 atmaero_control_p8_rad_intel FAIL Tries: 2
[RT] Error: Test 174 atmaero_control_p8_rad_micro_intel FAIL Tries: 2
[RT] Error: Test 175 regional_atmaq_intel FAIL Tries: 2
[RT] Error: Test 176 regional_atmaq_debug_intel FAIL Tries: 2
[RT] Error: Test 177 regional_atmaq_faster_intel FAIL Tries: 2
[RT] Error: Test 179 control_stochy_gnu FAIL Tries: 2
[RT] Error: Test 180 control_ras_gnu FAIL Tries: 2
[RT] Error: Test 182 control_flake_gnu FAIL Tries: 2
[RT] Error: Test 197 hrrr_gf_gnu FAIL Tries: 2
[RT] Error: Test 198 control_diag_debug_gnu FAIL Tries: 2
[RT] Error: Test 203 rap_noah_sfcdiff_cires_ugwp_debug_gnu FAIL Tries: 2
[RT] Error: Test 206 control_ras_debug_gnu FAIL Tries: 2
[RT] Error: Test 207 control_stochy_debug_gnu FAIL Tries: 2
[RT] Error: Test 221 conus13km_control_gnu FAIL Tries: 2
[RT] Error: Test 227 conus13km_debug_gnu FAIL Tries: 2
[RT] Error: Test 228 conus13km_debug_2threads_gnu FAIL Tries: 2
[RT] Error: Test 229 conus13km_radar_tten_debug_gnu FAIL Tries: 2
[RT] Error: Test 231 cpld_control_p8_gnu FAIL Tries: 2
[RT] Error: Test 232 cpld_control_nowave_noaero_p8_gnu FAIL Tries: 2
[RT] Error: Test 233 cpld_debug_p8_gnu FAIL Tries: 2
[RT] Error: Test 234 cpld_control_pdlib_p8_gnu FAIL Tries: 2
[RT] Error: Test 235 cpld_debug_pdlib_p8_gnu FAIL Tries: 2
[RT] Error: Test 236 datm_cdeps_control_cfsr_gnu FAIL Tries: 2
[RT] Log file shows failures.
[RT] Please obtain logs from /scratch1/BMC/gmtb/CCPP_regression_testing/NCAR_ufs-weather-model//run//1585301644/20240117014515/ufs-weather-model

**Comment by mkavulich** [comment]
Not sure how to parse this list of failures...it looks like all are expected to change? Interestingly it looks like a large number did pass though: 

Test 030 control_wrtGauss_netcdf_parallel_intel PASS
Test 056 regional_noquilt_intel PASS
Test 178 control_c48_gnu PASS
Test 181 control_p8_gnu PASS
Test 183 rap_control_gnu PASS
Test 184 rap_decomp_gnu PASS
Test 185 rap_2threads_gnu PASS
Test 186 rap_restart_gnu PASS
Test 187 rap_sfcdiff_gnu PASS
Test 188 rap_sfcdiff_decomp_gnu PASS
Test 189 rap_sfcdiff_restart_gnu PASS
Test 190 hrrr_control_gnu PASS
Test 191 hrrr_control_qr_gnu PASS
Test 192 hrrr_control_2threads_gnu PASS
Test 193 hrrr_control_decomp_gnu PASS
Test 194 hrrr_control_restart_gnu PASS
Test 195 hrrr_control_restart_qr_gnu PASS
Test 196 rrfs_v1beta_gnu PASS
Test 199 regional_debug_gnu PASS
Test 200 rap_control_debug_gnu PASS
Test 201 hrrr_control_debug_gnu PASS
Test 202 rap_diag_debug_gnu PASS
Test 204 rap_progcld_thompson_debug_gnu PASS
Test 205 rrfs_v1beta_debug_gnu PASS
Test 208 control_debug_p8_gnu PASS
Test 209 rap_flake_debug_gnu PASS
Test 210 rap_clm_lake_debug_gnu PASS
Test 211 control_wam_debug_gnu PASS
Test 212 rap_control_dyn32_phy32_gnu PASS
Test 213 hrrr_control_dyn32_phy32_gnu PASS
Test 214 hrrr_control_qr_dyn32_phy32_gnu PASS
Test 215 rap_2threads_dyn32_phy32_gnu PASS
Test 216 hrrr_control_2threads_dyn32_phy32_gnu PASS
Test 217 hrrr_control_decomp_dyn32_phy32_gnu PASS
Test 218 rap_restart_dyn32_phy32_gnu PASS
Test 219 hrrr_control_restart_dyn32_phy32_gnu PASS
Test 220 hrrr_control_restart_qr_dyn32_phy32_gnu PASS
Test 224 rap_control_dyn64_phy32_gnu PASS
Test 225 rap_control_debug_dyn32_phy32_gnu PASS
Test 226 hrrr_control_debug_dyn32_phy32_gnu PASS
Test 230 rap_control_dyn64_phy32_debug_gnu PASS

**Comment by mkavulich** [comment]
***Automated RT Failure Notification***
Machine: hera
Compiler: intel
Job: BL
[BL] Repo location: /scratch1/BMC/gmtb/CCPP_regression_testing/NCAR_ufs-weather-model//run//1585301644/20240117044515/ufs-weather-model
Please make changes and add the following label back: hera-intel-BL

**Comment by grantfirl** [comment]
> Not sure how to parse this list of failures...it looks like all are expected to change? Interestingly it looks like a large number did pass though:
> 
> Test 030 control_wrtGauss_netcdf_parallel_intel PASS Test 056 regional_noquilt_intel PASS Test 178 control_c48_gnu PASS Test 181 control_p8_gnu PASS Test 183 rap_control_gnu PASS Test 184 rap_decomp_gnu PASS Test 185 rap_2threads_gnu PASS Test 186 rap_restart_gnu PASS Test 187 rap_sfcdiff_gnu PASS Test 188 rap_sfcdiff_decomp_gnu PASS Test 189 rap_sfcdiff_restart_gnu PASS Test 190 hrrr_control_gnu PASS Test 191 hrrr_control_qr_gnu PASS Test 192 hrrr_control_2threads_gnu PASS Test 193 hrrr_control_decomp_gnu PASS Test 194 hrrr_control_restart_gnu PASS Test 195 hrrr_control_restart_qr_gnu PASS Test 196 rrfs_v1beta_gnu PASS Test 199 regional_debug_gnu PASS Test 200 rap_control_debug_gnu PASS Test 201 hrrr_control_debug_gnu PASS Test 202 rap_diag_debug_gnu PASS Test 204 rap_progcld_thompson_debug_gnu PASS Test 205 rrfs_v1beta_debug_gnu PASS Test 208 control_debug_p8_gnu PASS Test 209 rap_flake_debug_gnu PASS Test 210 rap_clm_lake_debug_gnu PASS Test 211 control_wam_debug_gnu PASS Test 212 rap_control_dyn32_phy32_gnu PASS Test 213 hrrr_control_dyn32_phy32_gnu PASS Test 214 hrrr_control_qr_dyn32_phy32_gnu PASS Test 215 rap_2threads_dyn32_phy32_gnu PASS Test 216 hrrr_control_2threads_dyn32_phy32_gnu PASS Test 217 hrrr_control_decomp_dyn32_phy32_gnu PASS Test 218 rap_restart_dyn32_phy32_gnu PASS Test 219 hrrr_control_restart_dyn32_phy32_gnu PASS Test 220 hrrr_control_restart_qr_dyn32_phy32_gnu PASS Test 224 rap_control_dyn64_phy32_gnu PASS Test 225 rap_control_debug_dyn32_phy32_gnu PASS Test 226 hrrr_control_debug_dyn32_phy32_gnu PASS Test 230 rap_control_dyn64_phy32_debug_gnu PASS
> 
> Starting the baseline generation now

Ya, me neither. At least a couple of the parent PRs were unspecific about the expected failures, so we really can't be that precise with this one. It looks like most of the passed tests were gnu, which is weird because the spack-stack changes that caused a lot of the failures were for both intel and gnu.

**Comment by grantfirl** [comment]
FYI, even though one of the PRs said that it required input changes, I think it was referring to copying data to wcoss only. I did do a quick comparison between the files in the WW3 input directory between those used for the ufs/dev branch and ours, and they were identical.

**Comment by grantfirl** [comment]
@mkavulich Since the RT failures are reasonable and the BL creation succeeded, I think that we're ready to start merging this one.

**Comment by grantfirl** [comment]
@mkavulich Please review and approve this PR as well as https://github.com/NCAR/fv3atm/pull/105, https://github.com/NCAR/ccpp-physics/pull/1035, and https://github.com/NCAR/ccpp-scm/pull/411

File: tests/machine/acorn [hunk 1]
```diff
@@ -1,5 +1,5 @@
   module load ecflow/5.6.0.13
```
Comment (mkavulich) [inline tests/machine/acorn:1]: I am assuming you applied these changes to the `machine` subdirectory files manually? Thanks for doing that, let me know in the future if you want help resolving testing conflicts until we can propose these changes back to the ufs-community repository

**Review by mkavulich — APPROVED** [review]
Looks good once submodules are updated

File: tests/machine/acorn (hunk 1 above)
Comment (grantfirl) [reply tests/machine/acorn:1]: For the NCAR files, ya, I think I did that. It was a while ago now.

**Comment by grantfirl** [comment]
Note: I forgot to update bl_date.ncar.conf. I'll add this to the next PR.

**Pull request merged**